Clockwork/Support/Vanilla/Clockwork.php

```diff
@@ -397,10 +402,10 @@ protected function configureSerializer()
 	public function configureShouldCollect()
 	{
 		$this->clockwork->shouldCollect([
-			'onDemand'        => $this->config['requests']['on_demand'],
-			'sample'          => $this->config['requests']['sample'],
-			'except'          => $this->config['requests']['except'],
-			'only'            => $this->config['requests']['only'],
+			'onDemand'       => $this->config['requests']['on_demand'],
+			'sample'         => $this->config['requests']['sample'],
+			'except'         => $this->config['requests']['except'],
+			'only'           => $this->config['requests']['only'],
```

Comment:
Is this change intentional?

Reply:
Yeah, just fixing some white-space. :)

Reply:
But this unfixed it, the last key `'exceptPreflight'` is now misaligned with the rest.

Reply:
Haha, the code editor I am using now, uses ligatures by default (fl in flight). Not a great idea in a code editor.

<img width="691" alt="Screenshot 2024-03-05 at 18 13 14" src="https://github.com/itsgoingd/clockwork/assets/821582/5562482e-e92a-4ff8-bd18-d86e4dbd3001">